pwiz_tools/examples/TextPSMReader.hpp

```diff
@@ -0,0 +1,171 @@
+
+#ifndef _TEXTPSMREADER_HPP_
+#define _TEXTSPMREADER_HPP_
+
+#include <string>
+#include <iostream>
+#include <map>
+#include <list>
+
+#include "pwiz/data/identdata/IdentData.hpp"
+#include "pwiz/data/common/CVTranslator.hpp"
+
+using namespace pwiz::data;
+using namespace pwiz::identdata;
+
+namespace pwiz::examples {
```

Comment:
Hah, apparently this syntax was added in C++17. I didn't know about it!

Reply:
Well, in that case I should dumb it down, right? should I nest instead?

namespace pwiz {
  namespace examples {
     ....
   }
}
???????

Reply:
Ah actually I was thinking about a global search and replace to switch all the files to the new format. I think we have C++17 as our minimum version. I could be wrong and it's at C++14 though, in which case this change would be for the best.

Reply:
OK, that's what I did. Don't want to be that guy that forces the need for a recent language feature. Seems to work.